### DISCUSSION TITLE
How to get GNU Assembler(GAS) instructions set ?

### DISCUSSION THREAD

**Discussion by hangshan-boop** [community/community#139477] [Programming Help]
### Body

Dear Mr/Mrs ! I access a pdf from gnu.org that gnu assembler(https://sourceware.org/binutils/docs/), but the document is not include GNU Assembler(GAS) instructions set.  I hope that you can help to find a pdf that it includeGNU Assembler(GAS) instructions set . Thank you! 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @hangshan-boop,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as their support portal. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by hangshan-boop** [comment]
ok. I recently read a book that operation system design and implement , meeting some problems how to design grep that x86-64. Now I hold source code that x-86, how to design x-64 grep. Thank you!



---Original---
From: ***@***.***&gt;
Date: Tue, Sep 24, 2024 02:57 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] How to get GNU Assembler(GAS) instructionsset ? (Discussion #139477)




 
Hi @hangshan-boop,
 
Thanks for being a part of the GitHub Community, we're glad you're here!
 
If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as their support portal. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.
 
Best of luck!

**Comment by hangshan-boop** [comment]
I have found the answer .



---Original---
From: ***@***.***&gt;
Date: Tue, Sep 24, 2024 02:57 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] How to get GNU Assembler(GAS) instructionsset ? (Discussion #139477)




 
Hi @hangshan-boop,
 
Thanks for being a part of the GitHub Community, we're glad you're here!
 
If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as their support portal. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.
 
Best of luck!
